packages/@okta/vuepress-site/docs/reference/api/asa/introduction/index.md

```diff
@@ -61,6 +62,14 @@ when retrieving lists:
 3. Check for a Link header value with `rel="next"`. If such a value exists,
    fetch it and repeat steps two and three.
 
+For resources that support pagination, specify the `offset` query parameter to fetch the list from a specific page cursor. Use the UUID string of the page cursor from the `Link` header. For example:
```

Comment:
it won't always be a uuid, but whatever is in the `next` Link header should be used here